src/images/hoth9-logo.svg

```diff
@@ -0,0 +1,159 @@
+<svg width="512" height="512" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You can remove this file. I'll be including it in my PR so you don't need to add it.